source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+   }
+
+You must specify the ``locale`` field in the collation; all other fields
+are optional. For a complete list of supported locales and the default values
+for the ``locale`` fields, see :manual:`Supported Languages and Locales
+</reference/collation-locales-defaults/#supported-languages-and-locales>`.
+For descriptions of each field, see the :manual:`Collation Document MongoDB
+manual entry </reference/collation/#collation-document>`.
+
+------------------
+Collation Examples
+------------------
+
+Set a Default Collation on a Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, we create a new collection called ``souvenirs`` on
+the ``sample_mflix`` database and assign a default collation with the
+"``fr_CA``" locale. The collation applies to all :manual:`operations that
+support collation </reference/collation/#operations-that-support-collation>`
+performed on that collection.
+
+.. literalinclude:: /code-snippets/collation/collection-collation.js
+   :language: javascript
+   :dedent: 4
+
+Assign a Collation to an Index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, we create a new index on the ``title`` field of
+the ``movies`` collection in the ``sample_mflix`` database with a collation
+set to the "``en_US``" locale.
+
+.. literalinclude:: /code-snippets/collation/index-collation.js
+   :language: javascript
+   :dedent: 4
+
+The following query uses the index we created:
+
+.. literalinclude:: /code-snippets/collation/query-index-collation.js
+   :language: javascript
+   :start-after: start query index collation
+   :end-before: end query index collation
+   :dedent: 4
+
+The following queries **do not** use the index that we created.  The first
+query does not include a collation and the second contains a different
+strength value than the collation on the index.
+
+.. literalinclude:: /code-snippets/collation/query-index-collation.js
+   :language: javascript
+   :start-after: start query without index collation
+   :end-before: end query without index collation
+   :dedent: 4
+
+Collation Query Examples
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Operations that read, update, and delete documents from a collection can use
+collations. This section includes examples of a selection of these. See the
+MongoDB manual for a full list of :manual:`operations that support collation
+</reference/collation/#operations-that-support-collation>`.
+
+find() and sort() Example
+`````````````````````````
+
+The following example calls both ``find()`` and ``sort()`` on a collection
+called ``contacts`` in a database called ``myproject`` using a German collation
+with the ``locale`` parameter set to ``de``.
+
+.. literalinclude:: /code-snippets/collation/find-sort-collation.js
+   :language: javascript
+   :dedent: 4
+
+findOneAndUpdate() Example
+``````````````````````````
+
+The following example calls the ``findOneAndUpdate()`` operation on a
+collection called ``names`` in which contains the following documents:
```

Comment:
It might be helpful to point out that this collection uses the default binary collation since that configuration was just discussed and does influence the behavior in the example.

Reply:
I'll specify this in each example since a reader might jump to this point from anywhere